### PR TITLE
Enable daily-release cluster rotation creation

### PR DESCRIPTION
**Pull request by chxchx**
**Release note**:

```release-note
none
```

### CODE REVIEW

File: scripts/update-e2e-cluster.sh [hunk 1]
```diff
@@ -53,8 +53,9 @@ while getopts :r:z:n: arg; do
   esac
 done
 
-if [ "${REPO}" != 'auth' ] && [ "${REPO}" != 'broker' ] && [ "${REPO}" != 'istio' ] && [ "${REPO}" != 'mixer' ] && [ "${REPO}" != 'pilot' ]; then
-  echo 'Must specific a repo and it must be auth/brokeristio/pilot/mixer'
+if [ "${REPO}" != 'auth' ] && [ "${REPO}" != 'broker' ] && [ "${REPO}" != 'istio' ] \
+  && [ "${REPO}" != 'mixer' ] && [ "${REPO}" != 'pilot' ] && [ "${REPO}" != 'daily-release' ]; then
+  echo 'Must specific a repo and it must be auth/brokeristio/pilot/mixer/daily-release'
```
Comment (rkpagadala) [inline scripts/update-e2e-cluster.sh]: Change brokeristio  to broker/istio

**Review by rkpagadala — APPROVED** [review]

File: scripts/update-e2e-cluster.sh (hunk 1 above)
Comment (chxchx) [reply scripts/update-e2e-cluster.sh]: Done

**Review by yutongz — APPROVED** [review]
/lgtm

**Pull request merged**